rllib/algorithms/ppo/torch/ppo_torch_rl_module.py

```diff
@@ -108,25 +75,20 @@ def setup(self) -> None:
         assert self.config.pi_config, "pi_config must be provided."
         assert self.config.vf_config, "vf_config must be provided."
 
-        if self.config.shared_encoder:
```

Comment:
I've removed the logic from here, as it is part of the configuration.

Reply:
super nice.